### PR TITLE
INN-3386: dev server ia nav

### PR DESCRIPTION
**Pull request by jacobheric**
## Description

New ia nav for dev server. 

https://github.com/user-attachments/assets/746521a6-e259-4270-a8cf-a7ef55550519

## Type of change (choose one)
- [x] Chore (refactors, upgrades, etc.)
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] Security fix (non-breaking change that fixes a potential vulnerability)
- [ ] Docs
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)

## Checklist
- [x] I've linked any associated issues to this PR.
- [x] I've tested my own changes.

*[Check our Pull Request Guidelines](https://github.com/inngest/inngest/blob/main/docs/PULL_REQUEST_GUIDELINES.md)*

### CODE REVIEW

File: ui/apps/dev-server-ui/src/components/Navigation/Manage.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    }),
+    pollingInterval: 1500,
+  });
+
+  return (
+    <div className={`jusity-center mt-5 flex flex-col`}>
+      {collapsed ? (
+        <div className="border-subtle mx-auto mb-1 w-6 border-b" />
+      ) : (
+        <div className="text-disabled leading-4.5 mx-2.5 mb-1 text-xs font-medium">Monitor</div>
```
Comment (djfarrelly) [inline ui/apps/dev-server-ui/src/components/Navigation/Manage.tsx:20]:
```suggestion
        <div className="text-disabled leading-4.5 mx-2.5 mb-1 text-xs font-medium">Manage</div>
```

**Review by djfarrelly — CHANGES_REQUESTED** [review]

File: ui/packages/components/src/Menu/MenuItem.tsx [hunk 2]
```diff
@@ -32,12 +37,23 @@ export const MenuItem = ({
             comingSoon
               ? 'text-disabled hover:bg-disabled cursor-not-allowed'
               : active
-              ? 'bg-secondary-4xSubtle text-info hover:bg-secondary-3xSubtle'
+              ? 'bg-secondary-3xSubtle text-info hover:bg-secondary-2xSubtle'
               : 'hover:bg-canvasSubtle text-muted'
```
Comment (djfarrelly) [inline ui/packages/components/src/Menu/MenuItem.tsx:41]: The design system has the hover state also make the text brighter. I think it needs something like this:
```suggestion
              : 'hover:bg-canvasSubtle text-muted hover:text-basis'
```

File: ui/apps/dev-server-ui/src/components/Navigation/Help.tsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+              className="text-subtle hover:bg-canvasSubtle mx-2 mt-2 flex h-8 cursor-pointer items-center px-2 text-[13px]"
+              value="docs"
+            >
+              <div className="hover:bg-canvasSubtle flex flex-row items-center justify-start">
+                <RiExternalLinkLine className="text-subtle mr-2 h-4 w-4 " />
+                <div>Inngest Documentation</div>
+              </div>
+            </Listbox.Option>
+          </Link>
+          <Link href="/support" target="_blank">
```
Comment (djfarrelly) [inline ui/apps/dev-server-ui/src/components/Navigation/Help.tsx:37]: Fix broken link:
```suggestion
          <Link href="https://app.inngest.com/support" target="_blank">
```

**Review by djfarrelly — CHANGES_REQUESTED** [review]

**Comment by jacobheric** [comment]
Ok, I updated this with most of the feedback here and in slack. Things not done yet pending feedback are:

* waiting for a new design token to land to update monitor/manage menu labels
* modal color updates for new design

File: ui/apps/dev-server-ui/src/app/(dashboard)/stream/Stream.tsx [hunk 4]
```diff
@@ -231,26 +232,35 @@ export default function Stream() {
...
+            <NewButton
+              kind="secondary"
+              appearance="outlined"
+              label={`${showInternalEvents ? 'Hide' : 'Show'} Internal Events`}
+              onClick={() => setShowInternalEvents((prev) => !prev)}
+            />
+            <NewButton
+              kind="secondary"
+              appearance="outlined"
+              label={freezeStream ? 'Resume Stream' : 'Freeze Stream'}
```
Comment (djfarrelly) [inline ui/apps/dev-server-ui/src/app/(dashboard)/stream/Stream.tsx:248]: I've long wanted to change this copy from "freeze." We don't use freeze anywhere, so I think leaning into "auto-refresh" as we will use elsewhere is better. 
```suggestion
              label={freezeStream ? 'Enable auto-refresh' : 'Disable auto-refresh'}
```

File: ui/apps/dev-server-ui/src/components/Event/SendEventModal.tsx [hunk 5]
```diff
@@ -200,12 +200,13 @@ export default function SendEventModal({ data, isOpen, onClose }: SendEventModal
...
-          <Button label="Cancel" appearance="outlined" btnAction={onClose} />
-          <Button
+        <div className="flex items-center justify-end gap-2">
+          <NewButton kind="secondary" label="Cancel" appearance="outlined" onClick={onClose} />
+          <NewButton
+            kind="primary"
             disabled={sendEventState.isLoading}
             label="Send Event"
-            btnAction={() => sendEvent()}
+            onClick={() => sendEvent()}
```
Comment (djfarrelly) [inline ui/apps/dev-server-ui/src/components/Event/SendEventModal.tsx:209]: oh thank goodness for `btnAction` being in the past!

**Review by djfarrelly — APPROVED** [review]

File: ui/apps/dev-server-ui/src/components/Event/SendEventModal.tsx (hunk 5 above)
Comment (jacobheric) [reply ui/apps/dev-server-ui/src/components/Event/SendEventModal.tsx:209]: 👍

**Review by djfarrelly — APPROVED** [review]

**Pull request merged**